### PR TITLE
Remove logging token

### PR DESCRIPTION
**Pull request by GSinseswa721**
## **PR Description**

 This PR removes the logging of sensitive information, specifically the orgToken from GraphQL request logs to improve data security.

## **Description of tasks that were expected to be completed**

- [x] Remove sensitive information from GraphQL request logs.
- [x] Implement logic to create a redacted copy of request variables.
- [x] Test  the log to  ensure that no sensitive information is exposed.

## **How has been this tested**

- Clone the repository.
- Checkout to `ft-remove-loggingToken` branch.
- Make sure all environment variables are configured according to the env.example file.
- Install the dependencies using `npm install`, then run `npm run` dev to verify that the application starts successfully.
- After logging in connected with the frontend login inputs include email, password and OrgToken are not going to be exposed.

## **Screenshots**

Screenshot (31)

### CODE REVIEW

File: src/helpers/sendEventEmails.ts
Comment (Yvan-David) [inline src/helpers/sendEventEmails.ts:1]: These changes are not necessary try to undo them

File: src/resolvers/invitationStatics.resolvers.ts
Comment (Yvan-David) [inline src/resolvers/invitationStatics.resolvers.ts:1]: even these changes also are not necessary

**Review by jniyonkuru — APPROVED** [review]

**Review by niyobertin — APPROVED** [review]

File: src/utils/logGraphQLRequests.ts [hunk 1]
```diff
@@ -6,12 +6,20 @@ const logGraphQLRequests = {
...
+      variables.orgToken = '[REDACTED]'
+    }
+
     logger.debug(
       'Request Query:\n' +
         noNewlinesQuery +
         '\n Request Variables: \n ' +
-        JSON.stringify(requestContext.request.variables)
+        JSON.stringify(variables)
     )
```
Comment (ncutixavier) [inline src/utils/logGraphQLRequests.ts]: Do we really need this logger at the moment? If not, why can't we remove?

**Review by Yvan-David — APPROVED** [review]

File: src/utils/logGraphQLRequests.ts [hunk 2]
```diff
@@ -6,12 +6,13 @@ const logGraphQLRequests = {
...
-        noNewlinesQuery +
-        '\n Request Variables: \n ' +
-        JSON.stringify(requestContext.request.variables)
-    )
+
+    // Redacted copy of the variables for logging purposes
+    const variables = { ...requestContext.request.variables }
+    if (variables?.orgToken) {
+      variables.orgToken = '[REDACTED]'
+    }
```
Comment (ncutixavier) [inline src/utils/logGraphQLRequests.ts:14]: Are we using this variable anywhere in the codebase? If not why don't we remove it?

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**